lib/elasticsearch/elastic-query-builder.js

```diff
@@ -488,6 +488,57 @@ class ElasticQueryBuilder {
     }
   }
 
+  buildPackedFieldClause (value, field) {
+    // Figure out the base property (e.g. 'owner')
+    const baseField = field.replace(/_packed$/, '')
+    // Allow supplied val to match against either id or value:
+    return {
+      bool: {
+        should: [
+          { term: { [`${baseField}.id`]: value } },
+          { term: { [`${baseField}.label`]: value } }
+        ]
+      }
+    }
+  }
+
+  buildMultiFieldClause (value, fields) {
+    return {
+      bool:
+        { should: fields.map(field => ({ term: { [field]: value } })) }
+    }
+  }
+
+  // This builds a filter cause from the value:
+  buildClause (value, field) {
+    const filterMatchesOnMoreThanOneField = field.length > 1
+    if (filterMatchesOnMoreThanOneField) {
+      return this.buildMultiFieldClause(value, field)
+    }
+    field = field[0]
+    const valueIsNotPackedValue = value.indexOf('||') < 0
+    const isPackedField = field.match(/_packed$/)
+    if (isPackedField && valueIsNotPackedValue) {
+      return this.buildPackedFieldClause(value, field)
```

Comment:
Again, doesn't need to happen as part of this PR, but we stand to simplify this thanks to your work to support multiple fields. If nothing in `FILTER_CONFIG` actually specifies a `_packed` field (opting instead to match on the related entity subfields as described above) we don't need this branching logic (i.e. the current `_packed` filters will just be handled by the multi-field handler above).